waveorder/io/reader.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+import os
+import zarr
+from tifffile import TiffFile
+
+
+class MicromanagerReader:
+
+    def __init__(self, folder):
```

Comment:
It will be best to turn off all print statements by default, and optionally add a verbose mode

Reply:
verbose mode now implemented